src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Avatar
+description: >
+  Avatars display typography, icons, or images in different sizes. They help users quickly recognize
+  an entity.
+package: '@zendeskgarden/react-avatars'
+propSheets:
+  - avatars/src/elements/Avatar.tsx
+---
+
+<Usage>
+  <div>
+    <MD isBold>When to use</MD>
```

Comment:
Use the `<strong>` element provided by the MarkdownProvider.

Reply:
Actually, if this is consistent between all usages, this can be an `MD isBold`, but it would be great if it could be abstracted into the component in some way along with the "When not to use"